js/store.js

```diff
@@ -360,6 +363,29 @@ const store = {
     // empty out request processing queue
     capture.queue = [];
     return await this.db.websites.clear();
+  },
+
+  async getOldestDate() {
```

Comment:
Perhaps something descriptive of what data is oldest would be better `getFirstRequestTime`?